spec/vast_verification.spec.js

```diff
@@ -0,0 +1,122 @@
+import { parserVerification } from '../src/parser/parser_verification';
+import { vastInLine } from './samples/inline_missing_values';
+import { vastMissingWrapperValues } from './samples/wrapper_missing_values';
+
+describe('parserVerification', function() {
+  describe('verifyRequiredValues', function() {
+    let warnings = [];
+
+    const parseXmlFromString = xmlString => {
```

Comment:
I would put this under `spec/utils/utils.js`

Reply:
I removed it, and replace it with the one already in the utils